code/modules/surgery/organs/stomach.dm

```diff
@@ -68,9 +68,9 @@
 			amount_max = max(amount_max - amount_food, 0)
 
 		// Transfer the amount of reagents based on volume with a min amount of 1u
-		var/amount = min(round(metabolism_efficiency * bit.volume, 0.1) + amount_min, amount_max)
+		var/amount = min((round(metabolism_efficiency * amount_max, 0.05) + rate_min) * delta_time, amount_max)
```

Comment:
Pretty confused about the logic here, why is a rate added onto the computed amount?

Also, why have you compensated all the metabolization_efficiency vars and not the rates? Feels like rates would change with DT but efficiencies should stay constant.

Reply:
Because what is confusingly called efficiency is actually how quickly the stomach transferrs the reagents to the mobs bloodstream. The max rate is a limiter to make sure we don't try to take more reagent out than there exists.

Reply:
Alright, should probably be renamed abit then, but not a job for this PR